### DISCUSSION TITLE
Approved but GitHub Education benefits not available

### DISCUSSION THREAD

**Discussion by Eight08dot** [community/community#139551] [GitHub Education]
### Hi everyone,

![Captura de pantalla 2024-09-23 211947](https://github.com/user-attachments/assets/9ba7d9ae-28ea-4750-b396-4cd3a52a906f)
i was approved weeks ago and i still dont have access to the benefits, does it takes that long to recieve them?

**Comment by davevad93** [comment]
Hi @Eight08dot ,

Given your status, you should have access to your benefits indeed. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "All offers" section and select one of the available offers, if you can redeem them that means your benefits are available, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue, support will help you solve it.
